src/Chainweb/Pact/RestAPI/Server.hs

```diff
@@ -323,6 +324,7 @@ spvHandler l cutR cid chainR (SpvRequest rk tid) = do
         { errBody = "Index lookup for hash failed: " <> sshow e }
       Right !i -> return i
 
+    tid <- chainIdFromText ptid
```

Comment:
this function shouldn't be used directly, because it doesn't check the chain id for validity. I'll take care of this in an upcoming PR.

Reply:
Should I add a TODO to this @larskuhtz?

Reply:
Discussed, and it won't get in his way on this side of things.